Sources/LLBBuildSystem/BuildKey.swift

```diff
@@ -17,7 +17,7 @@ public protocol LLBBuildKey: LLBKey {
 }
 
 /// Convenience protocol for referencing build system specific values.
-public protocol LLBBuildValue: LLBValue, LLBSerializable {}
+public protocol LLBBuildValue: LLBValue, LLBPolymorphicSerializable {}
```

Comment:
is this required? for it to be polymorphic serializable?

Reply:
hmm, let me try it with out it.   That may have been overzealous when fixing compile errors.

Reply:
Ah yes, this is required.   This is necessary for getting the auto synthesized conformance to CASObjectRepresentable.

Reply:
i.e. because of this:
```
extension LLBCASObjectRepresentable where Self: LLBPolymorphicSerializable {
    public func asCASObject() throws -> LLBCASObject {
        let any = try LLBAnySerializable(from: self)
        return LLBCASObject(refs: [], data: try any.toBytes())
    }
}
```